java/src/main/java/com/saucelabs/simplesauce/IEVersion.java

```diff
@@ -0,0 +1,6 @@
+package com.saucelabs.simplesauce;
+
+public enum IEVersion {
+    ;
```

Comment:
;

Reply:
Yea, unless I missed something obvious, this is how the compiler forces me to do it in Java 👎 @joshmgrant does that look right to you?